tests/wasm_sym/generate_symbolic_tests.sh

```diff
@@ -3,16 +3,20 @@
 rm -rf */
 touch __init__.py
 
-wget -nc -nv -O wabt.tgz -c https://github.com/WebAssembly/wabt/releases/download/1.0.12/wabt-1.0.12-linux.tar.gz
+if ! [ -x "$(command -v wast2json)" ]; then
+  wget -nc -nv -O wabt.tgz -c https://github.com/WebAssembly/wabt/releases/download/1.0.12/wabt-1.0.12-linux.tar.gz
+  tar --wildcards --strip=1 -xf wabt.tgz 'wabt-*/wast2json'
+  rm wabt.tgz
+else
+  cp "$(command -v wast2json)" .
+fi
+
 wget -nc -nv -O spec.zip -c https://github.com/WebAssembly/spec/archive/opam-1.1.zip
 
-yes | unzip -q -j spec.zip 'spec-*/test/core/*' -d .
-rm run.py README.md
+unzip -q -j spec.zip 'spec-*/test/core/*' -d .
+rm run.py README.md 
```

Comment:
Nit: Remove trailing whitespace
```suggestion
rm run.py README.md
```